src/pybind/mgr/dashboard/frontend/cypress/integration/cluster/create-cluster.po.ts

```diff
@@ -1,26 +1,19 @@
+import { ServicesPageHelper } from 'cypress/integration/cluster/services.po';
+
 import { PageHelper } from '../page-helper.po';
 import { NotificationSidebarPageHelper } from '../ui/notification.po';
+import { HostsPageHelper } from './hosts.po';
```

Comment:
We should avoid relative imports. In fact, a related PR exists for that:
https://github.com/ceph/ceph/pull/38065

Reply:
Done.